app/views/organisations/_organisations_list.html.erb

```diff
@@ -33,31 +33,40 @@
             },
           } do %>
             <% if @presented_organisations.ministerial_organisation?(organisation_type) %>
-              <%= render "govuk_publishing_components/components/organisation_logo", {
-                organisation: {
-                  name: organisation["logo"]["formatted_title"],
-                  url: organisation["href"],
-                  brand: organisation["brand"],
-                  crest: organisation["logo"]["crest"]
-                },
-                heading_level: 3,
-              } %>
+              <div class="govuk-!-margin-bottom-3">
+                <%= render "govuk_publishing_components/components/organisation_logo", {
+                  organisation: {
+                    name: organisation["logo"]["formatted_title"],
+                    url: organisation["href"],
+                    brand: organisation["brand"],
+                    crest: organisation["logo"]["crest"]
+                  },
+                  heading_level: 3,
+                } %>
+              </div>
             <% else %>
-              <%= link_to(organisation["title"], organisation["href"], class: "govuk-link organisation-list__item-title") %>
+              <div class="govuk-!-margin-bottom-3">
+                <h3 class="govuk-heading-s govuk-!-font-weight-regular govuk-!-margin-bottom-0 govuk-!-display-inline govuk-!-margin-right-2">
+                  <%= link_to(organisation["title"], organisation["href"], class: "govuk-link organisation-list__item-title") %>
+                </h3>
```

Comment:
_"Separate website"_ gets announced after the event for the user.  It ideally needs to be tied 
together to the user hears _"National Archives, separate website"_ 

![image](https://user-images.githubusercontent.com/71266765/152959465-752ca130-88a2-480d-98f7-95140a146796.png)

Reply:
I've just seen this - we should remove this entirely inline with guidance.

_[External links](https://design-system.service.gov.uk/styles/typography/#links)
If it’s an external link to a non-government website, make that clear in the link text. For example, ‘read advice on writing link text from [name of organisation]’. There’s no need to say explicitly that you’re linking to an external site. [Do not use an external link icon](https://designnotes.blog.gov.uk/2016/11/28/removing-the-external-link-icon-from-gov-uk/)._

Reply:
They're not even actual links to the external site... I think we should bring this to Adam's attention before going ahead though - might need to do comms around it first.

Reply:
Let's bundle this into the headings card to address separately from this PR.

Reply:
Okay, I won't make the changes now but have captured here: https://trello.com/c/jhnpLsS4/1160-document-outline-issues-on-collection-pages